### ISSUE TITLE
Fix 'test_feeds/test_download_feeds' tests in 4.2

### ISSUE THREAD

**Issue by damarisg** [wazuh/wazuh-qa#1542]
### Issue information

| Related issue |
|--|
| Closes https://github.com/wazuh/wazuh-qa/issues/1531|

When it comes to starting with the test fix, there is some information that can help you:
```
Module: Vulnerability Detector
Support Team:  "binary beasts"
Target: Manager   
OS: Linux
```
I add some information obtained while researched the logs that failed.

|  Type | Description  |
|--|--|
|Test Path | test_feeds/test_download_feeds
| Consistent | No 
| Test Execution | 2/3 executions failed
| Cases |   nvd, msu, json_redhat
| Cases Fails | 3 
| Summary | On `test_download_feeds` we detect that fail when called to `make_vuln_callback` because can't download provider in effect it can't see on the monitor log.
| Error message |  "The provider National Vulnerability Database download has taken more than 30 minutes"

In order to finish this issue the following tasks should be fulfilled:
- [x] Research of fails.
- [ ] Apply Fix on the Test.
- [ ] Full Green/ Full Yellow in `test_feeds/test_download_feeds` for 3 times. 
- [ ] Documentation of any important change done for these tests or the used tools.

**Comment by damarisg** [comment]
## 07-07-21
- Problems with the environment
- Launch Test to verify if the issue is correct.

## 08-07-21 
Working on NVD case:
- Launch test:
    - When I launch all cases Failed, It has problems verifying the message that we wait for, another problem is that the database is blocked when I try to download a lot of versions. (In this case from 2002)
    - Launched only fail case, and I obtain a PASS.
    - Launched only the fail case with another year (2016 and 2010)  and It works.
    
-   Launch manually:
   - Launch only failed case and I see that it work how we expected. it receives logs expected.
  - I will modify access to the database to see if it a problem here.

**Comment by mdengra** [comment]
### 2021-07-20

Used Wazuh-QA branch: `1531-full-yellow-vuln-det`
Test results with the modules `sca`, `syscollector` and `rootcheck` modules disabled in the `ossec.conf`: 

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_download_feeds_local_2020_dm_r1.log](https://github.com/wazuh/wazuh-qa/files/6846719/test_download_feeds_local_2020_dm_r1.log)| 2021-07-20 | Miguel | :yellow_circle:  
|[test_download_feeds_local_2020_dm_r2.log](https://github.com/wazuh/wazuh-qa/files/6846721/test_download_feeds_local_2020_dm_r2.log)| 2021-07-20 | Miguel | :yellow_circle:  
|[test_download_feeds_local_2020_dm_r3.log](https://github.com/wazuh/wazuh-qa/files/6846722/test_download_feeds_local_2020_dm_r3.log)| 2021-07-20 | Miguel | :yellow_circle:

**Comment by damarisg** [comment]
PR  #1617  changed the upgrade from the year.
This issue is blocked because of the continued inconsistency of [FileMonitor ](https://github.com/wazuh/wazuh-qa/issues/1602) issue.

Test Results after to merge:

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[YellowDownloadFeed.log](https://github.com/wazuh/wazuh-qa/files/6870090/YellowDownloadFeed.log)| 2021-07-23 | Seyla|  
|[YellowDownloadFeed2.log](https://github.com/wazuh/wazuh-qa/files/6870091/YellowDownloadFeed2.log)| 2021-07-23 | Seyla|  
|[YellowDownloadFeed3.log](https://github.com/wazuh/wazuh-qa/files/6870092/YellowDownloadFeed3.log)| 2021-07-23 | Seyla|

**Comment by damarisg** [comment]
This issue is closed because the test was disabled by #1602
